src/drive/mobile/lib/intents.js

```diff
@@ -130,8 +140,7 @@ export const intentHandlerIOS = store => async intent => {
         return blob
       } else {
         const dirEntry = await getEntry(item.data)
-        if (!dirEntry.type) dirEntry.type = getMimeTypeFromUTI(item.utis)
-        const file = await getFile(dirEntry)
+        const file = await getFile(dirEntry, getMimeTypeFromUTI(item.utis))
```

Comment:
Is `getMimeTypeFromUTI` a global function? I couldn't find any reference on it.

Reply:
No it's local to the file, I introduced it [in the previous commit](https://github.com/y-lohse/cozy-drive/blob/d945ff01fbc902e259cfa648655050fa4434688d/src/drive/mobile/lib/intents.js#L116-L132) ;)